views/js/qtiCommonRenderer/renderers/interactions/TextEntryInteraction.js

```diff
@@ -226,6 +234,15 @@ define([
     };
 
     var destroy = function destroy(interaction){
+
+        $("input.qti-interaction.qti-inlineInteraction.qti-textEntryInteraction").each(function(index, el) {
```

Comment:
Is it needed to have such a precise selector? Couldn't it wok with only `input.qti-textEntryInteraction`?

Reply:
Yes, it can!